modules/virt-connecting-service-ssh.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+// * virt/virtual_machines/vm_networking/virt-creating-service-vm.adoc
+
+:_content-type: PROCEDURE
+[id="virt-connecting-service-ssh_{context}"]
+= Connecting to a virtual machine by using SSH and a service
+
+You can connect to a virtual machine (VM) by using SSH and a service.
+
+.Prerequisites
+
+* You created a service to expose a VM.
+* You have an SSH client installed.
+* You are logged in to the cluster.
+
+.Procedure
+
+* Run the following command to access the VM:
++
+[source,terminal]
+----
+$ ssh <user_name>@<ip_address> -p <port> <1>
+----
+<1> Specify the cluster IP for a cluster IP service, the node IP for a node port service, or the dynamically allocated external IP address for a load balancer service.
```

Comment:
Instead of an IP address a load balancer can have a hostname too.

Reply:
For the sake of simplicity, I think it might be better not to mention FQDN until it is fully supported.